Code/LauncherUnified/Launcher.cpp

```diff
@@ -490,11 +513,15 @@ namespace O3DELauncher
             AZ::Interface<AZ::IConsole>::Get()->PerformCommand("sv_isDedicated false");
         }
 
-        bool remoteFileSystemEnabled{};
+        AZ::s64 remoteFileSystemEnabled{};
         AZ::SettingsRegistryMergeUtils::PlatformGet(*settingsRegistry, remoteFileSystemEnabled,
             AZ::SettingsRegistryMergeUtils::BootstrapSettingsRootKey, "remote_filesystem");
-        if (remoteFileSystemEnabled)
+        if (remoteFileSystemEnabled != 0)
```

Comment:
Yeah, I haven't done that change because all the other parameters in bootstrap are using 0/1, not false/true, so the C++ always always uses s64 (this one was a bug).
````
{
    "Amazon": {
        "AzCore": {
            "Bootstrap": {
                "remote_filesystem": 0,
                "provo_remote_filesystem": 0,
                "android_remote_filesystem": 1,
                "ios_remote_filesystem": 0,
                "mac_remote_filesystem": 0,
                "assets": "pc",
                "linux_assets": "linux",
                "android_assets": "android",
                "ios_assets": "ios",
                "mac_assets": "mac",
                "salem_assets": "salem",
                "allowed_list": "",
                "remote_ip": "127.0.0.1",
                "remote_port": 45643,
                "connect_to_remote": 0,
                "windows_connect_to_remote": 1,
                "linux_connect_to_remote": 1,
                "provo_connect_to_remote": 1,
                "salem_connect_to_remote": 0,
                "jasper_connect_to_remote": 0,
                "android_connect_to_remote": 1,
                "ios_connect_to_remote": 0,
                "mac_connect_to_remote": 0,
                "wait_for_connect": 0,
                "provo_wait_for_connect": 0,
                "salem_wait_for_connect": 0,
                "jasper_wait_for_connect": 0,
                "windows_wait_for_connect": 1,
                "linux_wait_for_connect": 1,
                "android_wait_for_connect": 1,
                "ios_wait_for_connect": 0,
                "mac_wait_for_connect": 0,
                "connect_ap_timeout" : 3,
                "launch_ap_timeout": 15,
                "wait_ap_ready_timeout": 1200
            }
        }
    }
}
````
Changing all these to use booleans can be done as a separate PR.

Reply:
> We should fix up all these platform_ entries and put them in proper /Platform/PlatformName/filename.setreg files. I'm not sure if we have a ticket for that work. Then deprecate the PlatformGet() registry functions.
> I know we can add user registry .setreg files and they will override engine .setreg files right? Do we recommend the user not modify the engine setreg files?

Yeah, that sounds something we can do in the future, although this can be part of a bigger refactor, because it'd be nice if all the setreg properties necessary to launch the app until the point it connects to AP would be separated from all the other setreg files. At the moment they are get baked together. Since the bootstrap properties always need to be deployed to the device, then when it connects to AP it can get all the other setreg from the PC and merge them. This way if a setreg property changes there is no need to redeploy assets, only if a bootstrap setreg is changed then that needs to be copied to the device.